packages/vanilla/package.json

```diff
@@ -18,7 +18,6 @@
   },
   "dependencies": {
     "@jsonforms/core": "^2.0.0-rc.1",
-    "@jsonforms/webcomponent": "^2.0.0-rc.1",
```

Comment:
must be a dev dependency at least, as otherwise the examples won't run as all the example still rely on the webcomponent and not react